tests/e2e/translations/new-language.wdio-spec.js

```diff
@@ -0,0 +1,64 @@
+const languagesPage = require('../../page-objects/display/languages.wdio.page');
+const commonPo = require('../../page-objects/common/common.wdio.page');
+const utils = require('../../utils');
+const userSettingsElements = require('../../page-objects/user-settings/user-settings.wdio.page');
+const contactsPage = require('../../page-objects/contacts/contacts.wdio.page');
+const reportsPage = require('../../page-objects/reports/reports.wdio.page');
+const loginPage = require('../../page-objects/login/login.wdio.page');
+const { expect } = require ('chai');
+
+describe('Adding new language', () => {
+  const addTranslations = async () => {
+    await utils.addTranslations('afr',{
+      'No messages found':'Geen boodskappe gevind nie',
+      'No contacts found':'Geen mense gevind nie',
+      'reports.none':'Geen verslae gevind nie',
+      'Analytics': 'Analytiks'
+    });
+    await browser.refresh();
+  };
+
+  before(async () => await loginPage.cookieLogin());
+
+  after(async () => {
```

Comment:
I don't think you need this anymore, because the browser session is reset between each suite.